components/src/core/selection-toolbar/selection-toolbar.component.scss

```diff
@@ -0,0 +1,40 @@
+.pxb-selection-toolbar-icon-wrapper {
+    display: flex;
+    align-items: center;
+    z-index: 1;
+    margin-right: 24px;
+    &.pxb-selection-toolbar-no-icon {
+        margin-right: 0;
+    }
+}
+
+.pxb-selection-toolbar-title-container {
+    display: flex;
+    padding-right: 16px;
+    justify-content: center;
+    align-self: stretch;
+    flex-direction: column;
+    width: calc(100% - 56px);
```

Comment:
why is this 100% -56 ? Shouldn't it only take up as much space as it needs? Or maybe just give it a `fxFlex` in the template?